halo2_proofs/src/plonk/prover.rs

```diff
@@ -34,8 +34,60 @@ where
     if circuits.len() != instances.len() {
         return Err(Error::Backend(ErrorBack::InvalidInstances));
     }
+    // This `create_proof` func is the legacy one, assuming that `compress_selectors: true`.
```

Comment:
instead of repeating the code here, you can just call
```rust
create_proof_custom(params, pk, true, circuits, instances, rng, transcript)
```

Reply:
Thanks a lot! @ed255 

Done https://github.com/privacy-scaling-explorations/halo2/pull/310/commits/9074fad671b75e775db455e52587069c9ac5d3b8